_commands/scripting/appendOperator.md

```diff
@@ -0,0 +1,68 @@
+---
+---
+
+\>\>
+-----
+
+'>>' is a shell operator that you can use to append the output in an existing file.
+
+~~~ bash
+$ echo line 1 >> example.txt
+$ cat example.txt
+line 1
+$ echo line 2 >> example.txt
+$ cat example.txt
+line 1
+line 2
+~~~
+
+<!--more-->
+
+### Useful Options / Examples
+
+*If the file does not exist, ">>" creates the file.
+
+~~~ bash
+$ echo line 1 >> example.txt
+$ cat example.txt
+line 1
+~~~
+
+*The appending operator allows for the appending of standard output and standard error to a file.
+
+  *Appending stdout to the file is appending the output of a file to the end of the other file.
+
```

Comment:
You've got to put a space after the `*` in order for the line to render as a bullet point.